src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/tasks-top-extra.jelly

```diff
@@ -24,16 +24,14 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:this="this">
-    <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
-    <j:set var="title" value="${%Folder Computation}"/> <!-- fallback, usually invisible -->
+
+    <j:set var="url" value="${h.getNearestAncestorUrl(request, it)}"/>
     <j:set var="computation" value="${it.computation}"/>
-    <j:if test="${computation != null}">
-        <j:set var="title" value="${computation.displayName}"/>
-    </j:if>
-    <l:task icon="images/24x24/notepad.png" href="${url}/computation/" title="${title}">
+
+    <l:task icon="images/24x24/notepad.png" href="${url}/${computation.searchUrl}" title="${computation.displayName}">
         <j:if test="${h.hasPermission(it,it.BUILD) and it.buildable}">
             <l:task icon="images/24x24/clock.png" href="${url}/build?delay=0" post="true" title="${%Run Now}"/>
         </j:if>
-        <l:task icon="images/24x24/terminal.png" href="${url}/computation/console" title="${%Log}"/>
+        <l:task icon="images/24x24/terminal.png" href="${url}/${computation.searchUrl}console" title="${%Log}"/>
```

Comment:
`href=console` It would have been enough since the `console` option only is available under `indexing/`.